config/locales/views/support/index.en.yml

```diff
@@ -30,3 +30,17 @@ en:
       <p><a href=%{link_lost_user}>%{link_lost_user}</a></p>"
       notice_pj_product: A screenshot can help us identify the element to improve.
       notice_pj_other: A screenshot can help us identify the issue.
+  supportadmin:
+    admin_intro_html: "<p>As an administration, you can contact us through this form. We'll answer you as quickly as possibly by e-mail or phone.</p>
+    <br>
+    <p><strong>Caution, this form is dedicated to public bodies only.</strong>
+    It does not concern individuals, companies nor associations (except those recognised of public utility). If you belong to one of these categories, contact us <a href=%{contact_path}>here</a>.</p>"
+    contact_team: Contact our team
+    pro_phone_number: Professional phone number (direct line)
+    pro_mail: Professional email address
+    admin demande rdv: I request an appointment for an online presentation of demarches-simplifiees.fr
+    admin question: I have a question about demarches-simplifiees.fr
```

Comment:
ça peut attendre une PR séparée (qui fera ça pour toute l'appli), mais ça vaudra le coup de [variabiliser](https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations) le nom d'appli dans les traductions:
```
# dans le yaml:
admin question: "I have a question about %{app_name}"
# dans la vue, un truc du genre:
t('admin email', scope: [:utils], app_name: APPLICATION_NAME)
```

Reply:
Oui, je me demande même si on ne pourrait pas monkey-patcher `I18n.lookup` pour que `%{app_name}` soit toujours une variable disponible, sans avoir à la spécifier dans le template…

Bon, mais en fait ça serait sale, et ça casserait tous les linters de `i18n-tasks` qui vérifient que les variables interpollées sont bien présentes.

En tout cas +1 pour le faire dans une PR suivante.